client/signup/config/flows.js

```diff
@@ -244,8 +244,8 @@ const flows = {
 };
 
 if ( config.isEnabled( 'signup/atomic-store-flow' ) ) {
-	flows[ 'atomic-store' ] = {
-		steps: [ 'design-type-with-atomic-store', 'themes', 'domains', 'plans-atomic-store', 'user' ],
+	flows[ 'store-nux' ] = {
+		steps: [ 'design-type-with-store-nux', 'themes', 'domains', 'plans-store-nux', 'user' ],
```

Comment:
I guess we can get rid of the steps that originate from this flow? And the resulting dead code?
@donpark @lamosty